migrations/emailmonitor/oxa_email_notifier.py

```diff
@@ -0,0 +1,239 @@
+    else:
+        os.system("echo '"+str(datetime.now())+" "+ log + ": "+str(error)+"' >> /oxa/oxa_email_notify.log")
+
+def GetActivationEmailForUser( activation_key ):
+    msg = "\r\nThank you for creating an account with Microsoft Learning!\r\n\r\nThere's just one more step before you can enroll in a course: you need to activate your Microsoft Learning account. To activate your account, click the following link. If that doesn't work, copy and paste the link into your browser's address bar.\r\n\r\n  https://openedx.microsoft.com/activate/"+activation_key+"\r\n\r\nIf you didn't create an account, you don't need to do anything. If you need assistance, please do not reply to this email message. Check the Support link at the bottom of the Microsoft Learning website.\r\n\r\n\r\n----\r\nMicrosoft respects your privacy. Please read our online Privacy Statement: http://go.microsoft.com/fwlink/?LinkId=521839\r\n\r\nThis is a mandatory service communication. To set your contact preferences for other communications, visit the Promotional Communications Manager: http://go.microsoft.com/fwlink/?LinkId=243191\r\n\r\nMicrosoft Corporation\r\nOne Microsoft Way\r\nRedmond, WA 98052 USA\r\n"
+
+    return msg
+
+def MarkDatabaseActivationEMailSent( row ):
+    db = MySQLdb.connect(IP,User,Password,Database)
+    cursor = db.cursor()
+
+    dt = datetime.now()
+    sql = "UPDATE oxa.oxa_activationfailed SET is_processed = 1, date_processed = '"+str(dt)+"' WHERE id="+str(row[0])
+    cursor.execute(sql)
+    sql = "UPDATE oxa.oxa_activationsummary SET resend = resend+1 WHERE xyear="+str(row[1])+" and xmonth="+str(row[2])+" and xday="+str(row[3])    
+    cursor.execute(sql)
+
+    db.commit()
+    cursor.close()
+    db.close()
+
+email_monitor_receivers_eng = ""
+email_monitor_receivers_sup = ""
+
+WriteLog("Started running process")
+dt = datetime.now()
+subject = dt.strftime("%Y-%m-%d %H:%M:%S")
+filename = dt.strftime("%Y%m%d%H%M%S") + ".mail.txt"
+f = open("/oxa/"+filename,"w")
+
+OutputMonthlySummary(f)
+OutputCurrentMonthSummary(f)
+OutputAllFailedEmailActivation(f)
+f.close()
+
+mailserver = None
+try:
+    mailserver = GetMailServerConnection()
+except Exception, ex:
+    WriteLog("Email Server Connection Error",ex)
+
+if mailserver != None:
+    try:
+        results = GetNotProcessedEmailActivationFailures() 
+        activated_emails = "Activation email sent to these users:\r\n\r\n"
+        activated_emails_keys = "Activation email sent to these users:\r\n\r\n"
+        activated_count = 0
+        for row in results:
+            msg = GetActivationEmailForUser(row[6])
+            try:
+                SendMail(mailserver,msg,row[5],"Activate Your Microsoft Learning Account")
+                MarkDatabaseActivationEMailSent(row)
+                WriteLog("Activation email is sent to user "+row[5]+" with key "+row[6])
+                activated_count = activated_count + 1
```

Comment:
I think ++ is more readable . edit: I remembered python is weird and doesn't support that. i'd still prefer += 1 which should work